wfl/Controllers/RestaurantsController.cs

```diff
@@ -2,13 +2,16 @@
 using System.Threading.Tasks;
 using Symphono.Wfl.Models;
 using Symphono.Wfl.Database;
-
+using Microsoft.Practices.Unity;
 
 namespace Symphono.Wfl.Controllers
 {
     [RoutePrefix("api/Restaurant")]
     public class RestaurantsController : ApiController
     {
+        IUnityContainer container = new UnityContainer();
```

Comment:
I thought it would be better to perform the Unity registration at startup but I don't know where that could take place.